jobs/schedulers.go

```diff
@@ -115,14 +117,15 @@ func (schedulers *Schedulers) Start() *Schedulers {
 					}
 				case newCfg := <-schedulers.configChanged:
 					for idx, scheduler := range schedulers.schedulers {
-						if !scheduler.Enabled(newCfg) {
+						if !schedulers.isLeader || !scheduler.Enabled(newCfg) {
```

Comment:
Because the `isLeader` boolean defaults to `true`, is it possible that we can get into this branch prior to the value being set by the `schedulers.clusterLeaderChanged` branch and then trigger a double-schedule?

Reply:
@mkraft - I would never say never, but I don't think so. The config loads prior to the ConfigListener being added. The app wouldn't be running yet, so I don't think it would be possible to trigger from the UI. But if you have a specific scenario in mind, we should explore it.